python/ray/tune/examples/tune_mnist_ray_mini.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python
```

Comment:
Is it necessary to add this new file? Can't we just pass some arguments into `tune_mnist_ray.py` to tell it to run very quickly?

Reply:
Ah, it was just because I didn't want to clutter the example too much, but on second thought makes sense to consolidate.